mobile/native/src/ln_dlc/mod.rs

```diff
@@ -153,14 +153,15 @@ pub fn run(data_dir: String, seed_dir: String) -> Result<()> {
         });
 
         // spawn a dedicated thread as this one might be CPU-heavy
-        std::thread::spawn({
-            let node = node.clone();
-            move || loop {
-                node.process_incoming_dlc_messages();
-
-                std::thread::sleep(PROCESS_INCOMING_MESSAGES_INTERVAL);
-            }
-        });
+        runtime
+            .spawn_blocking({
```

Comment:
it's not *that* bad, but it would take away one of the threads from our workers thread pool forever, so it's best to just spawn one ourselves (as I've done it later!)